docs/source/conf.py

```diff
@@ -154,16 +154,10 @@
 
 # Links matching with the following regular expressions will be ignored
 linkcheck_ignore = [
-    r"https://doi.org/10.1088/0034-4885/74/10/104401",
-    r"https://doi.org/10.1088/1367-2630/12/7/075008",
-    r"https://doi.org/10.7566/JPSJ.90.032001",
-    r"https://doi.org/10.7566/jpsj.90.032001",
-    r"https://dl.acm.org/doi/10.1145/3466752.3480059",
-    r"https://doi.org/10.1145/3466752.3480059",
-    r"https://doi.org/10.7566/jpsj.90.032001",
-    r"https://arxiv.org/abs/1612.02058",
-    r"https://arxiv.org/abs/1805.04492",
-    r"https://arxiv.org/abs/1807.05572",
+    r"https://arxiv.org/.*",
+    r"https://doi.org/.+",
+    r"https://link.aps.org/doi/.+",
+    r"https://www.sciencedirect.com/science/article/.+",
 ]
```

Comment:
What is the difference between `/` and `\` ?

Not requesting a suggested change, BTW. I am curious about `\d+/` used by [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-linkcheck_ignore).

Reply:
Oops, good catch! The `.` is a special character in regex and needs to be escaped with the `\`. Fixed.